Test Introductory Tutorial/services/com/vantiq/engines/EngineMonitor.json

```diff
@@ -37,10 +37,9 @@
   },
   "globalType" : null,
   "interface" : [ {
-    "description" : "given a temperature or speed event, determine the alert message (if any)",
```

Comment:
the descriptions were lost in this file. 
Can you add them both back?

Reply:
They weren't lost, they were intentionally moved.  I took this from the 1.35 update.  We should consult @dnhuch , but the change was intentional.

Reply:
Seems like the descriptions should be added back to that 1.35 update, too.  
The generateAlertMessage procedure itself didn't change, so I think the descriptions still apply.

Reply:
I'll let you and @dnhuch resolve that and then just do the same thing here...

Reply:
I didn't intentionally remove any descriptions when migrating the tutorial imports. But I think it makes sense to restore any places where there used to be descriptions and now there are not